contents/handbook/growth/marketing/index.md

```diff
@@ -4,20 +4,29 @@ sidebar: Handbook
 showTitle: true
 ---
 
-The Marketing & Content team exist to drive organic, word-of-mouth growth by:
-
-1. Producing world-class top-of-funnel content for our ICP
-2. Ensuring the information users need is easy to find and maintained to a high standard
-
-Marketing is the [newsroom](https://en.wikipedia.org/wiki/Newsroom) of PostHog. This means:
-
-- A small, talented team of in-house content creators with strong editorial capability – our default is to keep writing in-house, not farm it out to a network of freelancers.
- 
-- Building a reputation for high-quality, trustworthy content that is a go-to resource for our users and wider reading audience. 
-
-- Extreme responsiveness to opportunities, both internal and external (e.g. cool Hackathon projects, GDPR rulings).
- 
-- Proactively reporting and amplifying company and product news, rather than [artificially creating news](https://en.wikipedia.org/wiki/Tomorrow_Never_Dies) or buzz by telling people what to build.
+The marketing team has two core goals:
+
+1. Increase awareness of PostHog, especially among people in our ideal customer profile.
+2. Make engineers and PostHog users feel like badass ninjas by teaching them cool stuff.
+
+We do this by:
+
+- Building a reputation for world-class content
+- Having the coolest merch that people want to wear all the time
+- Constantly working to improve our documentation
+- Identifying where content can help users be more successful
+- Proactively promoting the work of product teams
+- Never telling them what to build
+- Producing ad campaigns that people actually like and share 
+- Holding a high bar for quality in everything we do
+- Never being satisfied with how high that bar is
+- Being weird, opinionated, and unafraid of being wrong
+- Reacting quickly to opportunities whenever they arise
+- Not being precious about our work and priorities
+- Having a team of talented, technically literate writers (and Andy)
+- Not overly relying on freelancers or guest contributors for content
+- Avoiding tedious enterprise marketing nonsense (PDFs, gated content, webinars, etc.)
+- Just being a great vibe generally
```

Comment:
This is a tiny bit 'how do you do fellow kids'

Reply:
I will remove if you feel strongly

Reply:
Yeah, I think if you are a great vibe you don't declare it, you just are it